source/utils/computeIntegrity.js

```diff
@@ -9,7 +9,7 @@ import crypto from 'crypto';
 /**
  * Compute SRI Integrity
  * @func computeIntegrity
- * See {@link https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity|Subresource Integrity} at MDN
+ * See {@link https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity Subresource Integrity} at MDN
```

Comment:
Also, could you update this link to `https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity`, as it's currently broken.

Reply:
The updated link works just fine for me:
![Capture](https://user-images.githubusercontent.com/34555510/110807779-5f6ef080-8251-11eb-88d4-31d4301fdb95.PNG)